backend/dataall/cdkproxy/stacks/pipeline.py

```diff
@@ -476,8 +510,8 @@ def write_deploy_buildspec(path, output_file):
                 - pip install -r requirements.txt
               build:
                 commands:
-                - aws sts get-caller-identity
-                - ddk deploy
+                    - aws sts get-caller-identity
+                    - ddk deploy
```

Comment:
indenting on these two lines seems off and may cause problems with the yaml parsing?